planet/clients/subscriptions.py

```diff
@@ -101,17 +59,17 @@ async def list_subscriptions(self,
             ClientError: on a client error.
         """
 
-        class _SubscriptionsPager(_Pager):
+        class _SubscriptionsPager(Paged):
```

Comment:
@jreiberkyle I accidentally pushed to the the wrong branch. I will undo this last commit now.